neon_utils/skills/mycroft_skill.py

```diff
@@ -61,13 +61,14 @@ def __init__(self, name=None, bus=None, use_settings=True):
                 os.path.abspath(sys.modules[self.__module__].__file__)))
             self.file_system = FileSystemAccess(os.path.join('skills', skill_id))
             self._settings = None
-            # TODO: Should below be global config? This implementation is designed
-            #       to allow skills to specify overrides per-skill
-            self._speak_timeout = 30
-            self._get_response_timeout = 15  # 10 for listener, 5 for STT, then timeout
 
         super(PatchedMycroftSkill, self).__init__(name, bus, use_settings)
         self.gui = SkillGUI(self)
+        # TODO: Should below be global config? This implementation is designed
+        #       to allow skills to specify overrides per-skill
+        self._speak_timeout = 30
+        self._get_response_timeout = 15  # 10 for listener, 5 for STT, then timeout
```

Comment:
Can those also be in config for each skill separately?

Reply:
The idea here is that it's up to the skill, so in the skill's `initialize` method, you could do something like:

```python
def initialize(self):
    self._get_response_timeout = self.settings.get("response_timeout") or self._get_response_timeout
```